module-gui/gui/widgets/ListViewEngine.hpp

```diff
@@ -109,7 +109,7 @@ namespace gui
         [[nodiscard]] unsigned getFocusItemIndex();
 
         /// Total provider elements count
-        unsigned int elementsCount = listview::nPos;
+        unsigned elementsCount = listview::nPos;
```

Comment:
`auto` would be even better here 🙂

Reply:
Unfortunately to declare class member with `auto` it needs to be also `static` - I removed this `int` only to unify unsigned integers declarations across the whole file, as all other declarations here omit it.

Reply:
Ah, just noticed the context! Too many things at the same time, sorry :)